src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CcdCaseAssignment.java

```diff
@@ -91,4 +131,168 @@ private HttpHeaders createHeaders(String serviceAuthorizationToken, String acces
         headers.set(SERVICE_AUTHORIZATION, serviceAuthorizationToken);
         return headers;
     }
-}
\ No newline at end of file
+
+    private void addRespondentRepresentativeToMultiple(CaseDetails caseDetails) throws IOException {
+        if (!YES.equals(caseDetails.getCaseData().getMultipleFlag())) {
+            return;
+        }
+
+        String adminUserToken = adminUserService.getAdminUserToken();
+        String caseId = caseDetails.getCaseId();
+        String userToAddId = getEventTriggerUserId(adminUserToken, caseId);
+
+        if (userToAddId.isEmpty()) {
+            log.info("Add Respondent Representative to Multiple failed. Legal Rep Id not found for case {}", caseId);
+            return;
+        }
+
+        String caseType = caseDetails.getCaseTypeId() + "_Multiple";
```

Comment:
Feel like we might be doing something similar in other places. Might be a good idea to move to a helper